app/services/activity_defaults.rb

```diff
@@ -49,6 +50,14 @@ def extending_organisation
     delivery_partner_organisation
   end
 
+  def originating_report
+    Report.where(
```

Comment:
Could we do `find_by` here, rather than `where...first`?

Reply:
yes, for sure, thks for that.